backend/src/resolvers/comments.js

```diff
@@ -31,15 +11,15 @@ export default {
         throw new UserInputError(`Comment must be at least ${COMMENT_MIN_LENGTH} character long!`)
       }
       const { postId } = params
-      delete params.postId
       const comment = await neo4jgraphql(object, params, context, resolveInfo, false)
 
       const session = context.driver.session()
 
       await session.run(`
-          MATCH (post:Post {id: $postId}), (comment:Comment {id: $commentId})
-          MERGE (post)<-[:COMMENTS]-(comment)
-          RETURN comment {.id, .content}`, {
+        MATCH (post:Post {id: $postId}), (comment:Comment {id: $commentId}), (author:User {id: $userId})
```

Comment:
I think that it is! 
I found the idea on neo4j's docs https://neo4j.com/docs/cypher-manual/current/clauses/merge/#merge-merge-on-multiple-relationships

I am very happy to have found the error in my backend test because before I was running two sessions to associate the posts and author. I think this could be a performance issue(?) 

We need to write some tests that lead to the implementation  where a comment cannot be created without a post and author.